tests/e2e/conftest.py

```diff
@@ -14,7 +16,7 @@
 from typing import Dict, Generator, Optional
 
 import pytest
-from e2e.utils import helpers, infrastructure
+from e2e.utils import infrastructure
```

Comment:
should we merge this  line with line 9?

Reply:
YES! auto-refactor from PyCharm when moving things around.